src/components/FacilityList.js

```diff
@@ -22,21 +24,27 @@ export default function FacilityList(props) {
   }, []);
 
   return (
-    <styledFacilityList className="inmate-list">
-      {facilities.map(facility => (
+    <StyledFacilityList className="inmate-list">
+      {facilities.map((facility, index) => (
         <Facility
-          key={facility.id}
+          key={facilities[index]}
           id={facility.id}
           Prison_Name={facility.Prison_Name}
           Location={facility.Location}
           available_prisoners={facility.available_prisoners}
         />
       ))}
-    </styledFacilityList>
+      <Link to="/addPrison">
+        <Button  className="add-prison">
```

Comment:
className isn't necessary for Button, instead add `type="primary"`
There's a lot more you can do with the button. here's a link to antd for reference: https://ant.design/components/button/